src/components/layouts/container.ts

```diff
@@ -0,0 +1,5 @@
+import tw from 'tailwind-styled-components';
```

Comment:
the container should be one of many exports of `layouts.ts`

Reply:
I just thought we could keep the different layouts separately for easy read and identification.